src/components/ThoughtRender.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from "react";
+import PropTypes from "prop-types";
+import Lottie from "lottie-react";
+import loadingAnimation from "../assets/loadingAnimation.json";
+
+export const ThoughtRender = ({ setThoughts, loading, setLoading }) => {
+  const URL = "https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts";
+
+  useEffect(() => {
+    const loadingTime = setTimeout(() => {
+      fetch(URL)
+        .then((result) => {
+          if (!result.ok) {
+            throw new Error("Network is bad. Please reload the Page.");
+          }
+          return result.json();
+        })
+        .then((data) => {
+          setThoughts(data);
+          setLoading(false);
+        })
+        .catch((error) => {
+          console.error("Error fetching the data:", error);
+          setLoading(false);
+        });
+    }, 4000);
```

Comment:
Aah! You have a standard delay of 4 sek? :D I was wondering how come it took some time to load. Is it for the animation to show properly?